File: visual-model-database/src/main/java/org/visual/model/database/driver/Driver.java
package org.visual.model.database.driver;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import org.eclipse.aether.artifact.Artifact;
import org.eclipse.aether.artifact.DefaultArtifact;

@RequiredArgsConstructor
@Getter
public enum Driver {
    MYSQL(new DefaultArtifact("", "", "", "")),

    POSTGRESQL(new DefaultArtifact("", "", "", ""));

    private final Artifact artifact;
}

File: src/main/kotlin/org/visual/driver/Driver.kt
/* (C)2024*/
package org.visual.driver

import lombok.Getter
import lombok.RequiredArgsConstructor
import org.eclipse.aether.artifact.Artifact
import org.eclipse.aether.artifact.DefaultArtifact

@RequiredArgsConstructor
@Getter
enum class Driver(val artifact: Artifact) {
  MYSQL(DefaultArtifact("", "", "", "")),
  POSTGRESQL(DefaultArtifact("", "", "", ""))
}
